cli/api/options.go

```diff
@@ -246,6 +247,7 @@ func GetDefaultOptions(config utils.ConfigReader) Options {
 		DockerLogDriver:      config.StringVal("DOCKER_LOG_DRIVER", "json-file"),
 		DockerLogConfigList:  config.StringSlice("DOCKER_LOG_CONFIG", []string{"max-file=5", "max-size=10m"}),
 		AllowLoopBack:        strconv.FormatBool(config.BoolVal("ALLOW_LOOP_BACK", false)),
+		UIPollFrequency:      config.IntVal("UI_POLL_FREQUENCY", 3000),
```

Comment:
Are the units really in milliseconds?  Shouldn't they be in something more user-friendly like seconds?